src/components/avatars/OcAvatarGuest.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <oc-avatar-item
+    icon="user_remote"
+    icon-color="#D78841"
+    :name="name"
+    :accessible-label="accessibleLabel"
+  ></oc-avatar-item>
+</template>
+
+<script>
+export default {
+  name: "OcAvatarGuest",
+  status: "ready",
+  release: "2.1.0",
```

Comment:
Please set `status` to `review` and release to an empty value. Will be updated during release.